internal/awstesting/sandbox/Dockerfile.test.go1.13

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
```

Comment:
Hey, I just stumbled across this PR and was curious to take a look.  The two consecutive `FROM` lines don't work the way I think you think they work; the `golang:1.13` image is not layered on top of the `ubuntu:18.04` image but rather replaces it.  (Multiple `FROM` lines are useful in multi-stage builds where artifacts are constructed in one container and copied to another.)  If you run the image that results from this Dockerfile, you'll see that it's based on Debian Buster instead of Ubuntu.

```
$ docker run --rm internal/aws-sdk-go-v2:1.13 cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

Compare to:

```
$ docker run --rm ubuntu:18.04 cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.1 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

If you want to test compiling and run your unit tests on Ubuntu, you'd need to use the Ubuntu base image and install the Go compiler into it.

Reply:
Thanks for pointing this out! We'll fixup the Dockerfiles to use the correct syntax. I don't think we explicitly need testing on specific Os versions in these tests. At least not yet. The base Debian images should be sufficient.